library/src/animationManager.cxx

```diff
@@ -113,12 +119,10 @@ void animationManager::Initialize()
       int nbTimeSteps;
       vtkNew<vtkDoubleArray> timeSteps;
 
-      // Discard timesteps, F3D only cares about real elapsed time using time range
-      // Specifying the frame rate in the next call is not needed after VTK 9.2.20230603 :
+      // Discard timesteps, F3D only cares about elapsed time using time range and deltaTime
+      // Specifying a frame rate (60) in the next call is not needed after VTK 9.2.20230603 :
       // VTK_VERSION_CHECK(9, 2, 20230603)
-      double frameRate = this->Options.scene.animation.frame_rate;
-      this->Importer->GetTemporalInformation(
-        animIndex, frameRate, nbTimeSteps, timeRange, timeSteps);
+      this->Importer->GetTemporalInformation(animIndex, 60, nbTimeSteps, timeRange, timeSteps);
```

Comment:
Don't you need `1.0 / this->DeltaTime` instead of `60`?

Reply:
this parameter has no effect at all, see the comment above

Reply:
So let's put `-1` and remove the comment if we do not support VTK <= 9.2.20230603

Reply:
but we do support vtk 9.2.6, which require framerate > 0 to provide temporal information.

In any case, i should not assume how its implemented in other importers, so Ill use the correct framerate.

Reply:
Well I cant, because this is during initialization.

I can either:

 - Use -1 and drop support for VTK  9.2.6 -> Nope
 - Keep 60 as a reasonnable value for the 9.2.6 API
 - Put a higher arbitrary value ?

In any case I do not think it matters much. Importers based on VTK > 9.2.6 should support the correst API (no framerate needed). Using 60 here juste means VTK generate 60 timesteps, that we do not use, ever.

Reply:
Not really important, let's do what you think is better.